lumicks/pylake/population/mixture.py

```diff
@@ -121,6 +122,26 @@ def label(self, trace):
         output_states = np.argsort(self._map)  # output model state labels in wrapped model order
         return output_states[labels]  # output model labels
 
+    def dwell_times(self, statepath, dt_seconds, exclude_ambiguous_dwells):
```

Comment:
Why is this a method? You don't use anything contained in `self`.

The fact that it doesn't, makes me wonder about the API though. Why not take a slice, then you can label it for the user, extract the time step etc?

Also, would it make sense to make `exclude_ambiguous_dwells` a keyword argument?

Reply:
yeah you're right. I've changed it now and I'll add a section in the docs for how to use this with `DwelltimeModel` in the next PR